node/pkg/logscribe/logprocessor/logprocessor.go

```diff
@@ -141,15 +144,55 @@ func hashLog(log LogInsertModel) string {
 	return hex.EncodeToString(hash.Sum(nil))
 }
 
+func (p *LogProcessor) fetchCurrentIssues(ctx context.Context) ([]string, error) {
+	opts := &github.IssueListByRepoOptions{
+		State: "open",
```

Comment:
would be nice if it can be filtered by the issue creater, the owner of the github token